docker-compose.yaml

```diff
@@ -0,0 +1,89 @@
+version: "3"
+
+services:
+  orodha-gateway:
+    hostname: orodha-gateway
+    container_name: orodha-gateway
+    image: nginx:latest
+    volumes:
+      - ./config/gateway/conf.d:/etc/nginx/conf.d
+    ports:
+      - "80:80"
+    networks:
+      - orodha-network
+
+  orodha-user-server:
+    hostname: user-service-server
+    container_name: user-service
+    image: edisonstuart/orodha-user-service:latest
+    networks:
+      - orodha-network
+
+  orodha-user-mongo:
+    hostname: user-service-mongo
+    container_name: user-service-mongo
+    image: mongo
+    restart: always
+    volumes:
+      - ./scripts/user_scripts:/docker-entrypoint-initdb.d
+    environment:
+      MONGO_INITDB_DATABASE: admin
+      USER_MONGO_INITDB_ROOT_USERNAME: $USERDBROOTUSER
+      USER_MONGO_INITDB_ROOT_PASSWORD: $USERDBROOTPASSWORD
+      USER_DBUSER: $USERDBUSER
+      USER_DBPASSWORD: $USERDBPASSWORD
+      USER_DBNAME: $USERDBNAME
+    networks:
+      - orodha-network
+
+  orodha-list-server:
+    hostname: list-service-server
+    container_name: list-service
+    image: edisonstuart/orodha-list-service:latest
+    networks:
+      - orodha-network
+
+  orodha-list-mongo:
+    hostname: list-service-mongo
+    container_name: list-service-mongo
+    image: mongo
+    restart: always
+    volumes:
+      - ./scripts/list_scripts:/docker-entrypoint-initdb.d
+    environment:
+      MONGO_INITDB_DATABASE: admin
+      LIST_MONGO_INITDB_ROOT_USERNAME: $LISTDBROOTUSER
+      LIST_MONGO_INITDB_ROOT_PASSWORD: $LISTDBROOTPASSWORD
+      LIST_DBUSER: $LISTDBUSER
+      LIST_DBPASSWORD: $LISTDBPASSWORD
+      LIST_DBNAME: $LISTDBNAME
+    networks:
+      - orodha-network
+
+  orodha-notification-server:
+    hostname: notification-service-server
+    container_name: notification-service
+    image: edisonstuart/orodha-notification-service:latest
+    networks:
+      - orodha-network
+
+  orodha-notification-mongo:
+    hostname: notification-service-mongo
+    container_name: notification-service-mongo
+    image: mongo
+    restart: always
+    volumes:
+      - ./scripts/notification_scripts:/docker-entrypoint-initdb.d
```

Comment:
@Edison-Stuart You're going to want all of the DB scripts in one folder; as it's currently set up, only the notification service DB will get set up. Create `/scripts/db-init` and house all of the init scripts there, mounting `/scripts/db-init` into the entrypoint dir in the mongo container. The env variables can be sent in using a local file, some `orodha.env`, you can use that as the env for the deployment with `docker-compose --env-file`. This way, you can send only that config which is relevant into each container.